docs/DeveloperGuide.md

```diff
@@ -659,6 +659,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Edit a client's details
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to view list
```

Comment:
Let's standardise ending all the sentences with a full stop. I've edited the previous use cases to do so too in #364